phonenumbers.go

```diff
@@ -984,14 +980,21 @@ func normalizeHelper(number string,
 	return normalizedNumber.String()
 }
 
-// Convenience method to get a list of what regions the library has metadata for.
-func GetSupportedRegions() map[string]struct{} {
+// GetSupportedRegions returns all regions the library has metadata for.
+func GetSupportedRegions() map[string]bool {
```

Comment:
Note this is a change to the interface we expose but I think in usage that is fine. (and I much prefer exposing sets as `map[]bool` rather than `map[]struct`.

Reply:
I personally don't mind the change in the interface as it is minor. About exposing bool vs struct{} I guess it's just a matter of preference:

PROS: bool provides a slightly better experience when checking for existence in the map
CONS: a little extra memory for storing the bool values

I found myself preferring bools on some occasions too, so I totally understand.

Reply:
Ya, influenced a bit by this:
  https://stackoverflow.com/questions/37320287/maptstruct-and-maptbool-in-golang

In general, it seems more obvious for one, easier to use for two, and in this case these are small enough maps that it is seriously in the noise.